tests/product/test_server_install.py

```diff
@@ -227,24 +227,6 @@ def test_install(self, installer=None):
             self.assert_has_default_config(container)
             self.assert_has_default_connector(container)
 
-    def test_install_worker_is_pa_master(self):
```

Comment:
I agree, testing without explicitly setting JAVA8_HOME is implicitly tested in all the other tests so we can remove `test_install()`.